From 99e430a056d9c1166b4d9681e3d3be98880f680f Mon Sep 17 00:00:00 2001
From: Ken Ma <make@marvell.com>
Date: Wed, 20 Apr 2016 23:51:52 +0800
Subject: [PATCH 1085/1240] config: a3700: enable driver model

- This patch is the preparation for generic gpio
  driver support, which follows the new convention
  of the driver model.
- Since none of the driver in our legacy SoC
  supports uclass model, there is no impacts on
  these driver at all.

Change-Id: I67432d4fa81ac800acf7f5fc862e9ddabb99c779
Signed-off-by: Ken Ma <make@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/29231
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Wilson Ding <dingwei@marvell.com>
Reviewed-by: Haim Boot <hayim@marvell.com>
---
 configs/mvebu_armada3700_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/mvebu_armada3700_defconfig b/configs/mvebu_armada3700_defconfig
index 74a26a5..39b7a33 100644
--- a/configs/mvebu_armada3700_defconfig
+++ b/configs/mvebu_armada3700_defconfig
@@ -27,6 +27,7 @@ CONFIG_CMD_MVEBU_HW_INFO=y
 +S:CONFIG_OF_CONTROL=y
 +S:CONFIG_OF_EMBED=y
 +S:CONFIG_DEFAULT_DEVICE_TREE="armada-3700-db"
+CONFIG_DM=y
 CONFIG_MVEBU_ADVK_PCIE=y
 CONFIG_MV_INCLUDE_SATA=y
 CONFIG_SCSI_AHCI_PLAT=y
-- 
1.9.1

